datadog/resource_datadog_synthetics_test_.go

```diff
@@ -323,39 +328,39 @@ func updateSyntheticsTestLocalState(d *schema.ResourceData, syntheticsTest *data
 	d.Set("type", syntheticsTest.GetType())
 
 	actualRequest := syntheticsTest.GetConfig().Request
-	localRequest := make(map[string]string)
+	localRequest := schema.ResourceData{}
```

Comment:
Why considering the request as a resource itself?

Reply:
Intention was to avoid converting to string, but this does not pass acceptance tests, so reverting.